### PR TITLE
Add TOC and edit UG and DG

### LINKED ISSUE #213: [PE-D][Tester C] The list view of schedules is not in chronological order
Although it's not explicitly mentioned in the UG that the schedule found in the list view has to be in chronological order, it can be quite inconvenient for the user to view their schedule if not ordered properly.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: lokidoki102/ped#9

### PR DESCRIPTION
**Pull request by cheahTJ**
Resolve #213, #216, #221

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -103,16 +125,16 @@ Let us get started!
...
+### **Notes about the command format:**
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  > `delete id/NUSID`, `NUSID` is a parameter which need to be supplied.
+  > In the delete command as such: `delete id/NUSID`, `NUSID` is a parameter which need to be supplied.
 
 * Items in square brackets are optional.<br>
   > `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or simply as `n/John Doe` without specifying the `TAG` as it is optional.
 
 * Items with `…`​ after them can be used multiple times or zero times.<br>
```
Comment (hjuntan) [inline docs/UserGuide.md:136]: Can you update the multiplicity to one or more as for the `group id/NUSID...` allows multiple people to be grouped but need at least one NUSID provided?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -46,7 +69,7 @@ Let us get started!
 
 1. Copy the file to the folder you want to use as the _home folder_ for AronaPro in your desired location. For demonstration purpose, we will save the downloaded jar file into `test` folder and saved it to `Desktop` on either Mac or Windows machines. 
 
-1. Open a command terminal, either by typing `terminal` on Mac or `PowerShell` on windows and perform the following steps to reach the `test` folder created earlier in order to run the application:
+1. Open a command terminal, either by typing `terminal` on Mac or `PowerShell` on Windows machine and perform the following steps to reach the `test` folder created earlier in order to run the application:
```
Comment (RingoftheKing) [inline docs/UserGuide.md:72]: Interesting specification, but what's the diff LOL

File: docs/UserGuide.md [hunk 3]
```diff
@@ -139,8 +162,21 @@ Adds a person to AronaPro.
...
+> * `EMAIL` can be duplicated for different persons.
+> * `EMAIL` follows the following format and constraints:
+  >   * It should have a local-part followed by a `@` and then a domain name.
+>     * Local-part should only contain alphanumeric characters and special characters(+ _ . -).
+>     * Local-part may not start or end with any special characters.
+>     * Domain name is made up  of domain label separated by periods.
+  >     * Domain name must end with a domain label at least 2 characters long.
+>       * Have each domain label start and end with alphanumeric characters.
+>       * Have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
+> * For `TAG``, the valid forms have to be either 1 of these:  Professor, TA, Student, None.
```
Comment (RingoftheKing) [inline docs/UserGuide.md:179]: Might need to specify (Case-sensitive)
If not already done elsewhere. 
(Edit: already specified below lol, but ig if you want you can sneakily save one known issue by clearly stating here that it's case sensitive)

**Comment by RingoftheKing** [comment]
lgtm

**Pull request merged**